### ISSUE TITLE
Singel machine vs multi-machine validation accuracy

### ISSUE THREAD

**Issue by sakh251** [richardkxu/distributed-pytorch#5]
Hi,

I ran the code on single machine with 3 gpus and 2 machines with 6 gpus (each 3).
The validation accuracy of single machine is 3 percent more than multi-machine.
So I can not reproduce the reported results.

Is there any clue to found out the problem.
